api/transport/outbound.go

```diff
@@ -62,6 +64,31 @@ type UnaryOutbound interface {
 	Call(ctx context.Context, request *Request) (*Response, error)
 }
 
+type UnchainedUnaryOutbound interface {
+	Outbound
+
+	// UnchainedCall is called without interceptor.
+	UnchainedCall(ctx context.Context, request *Request) (*Response, error)
+}
+
+// UnchainedOnewayOutbound defines a transport outbound for oneway requests
+// that does not involve any interceptors.
+type UnchainedOnewayOutbound interface {
+	Outbound
+
+	// UnchainedOnewayCall is called without interceptor.
+	UnchainedOnewayCall(ctx context.Context, request *Request) (Ack, error)
```

Comment:
Method name should be in verb form conventionally  - so I suggest to use `UnchainedCallOneway` or `CallOnewayUnchained`.